apps/full-site-editing/full-site-editing-plugin/block-inserter-modifications/index.js

```diff
@@ -0,0 +1,5 @@
+/**
+ * Internal dependencies
+ */
+import './new-blocks-showcase';
+import './tips';
```

Comment:
Let's not combine those into a single index file but rather keep them separate and add a build command for it.